NebulaClient/MultiplayerClientSession.cs

```diff
@@ -111,6 +135,12 @@ public void Reconnect()
             ConnectInternal();
         }
 
+        public void UpdatePingIndicator()
+        {
+            float delay = (Time.time - pingTimestamp);
+            pingIndicator.text = $"Ping: {(int)(delay*1000)}ms";
```

Comment:
Only update if it changed?

Reply:
This should be updated every second, so I do not think there is much impact. But I will add that condition.

Reply:
done :)